web/migrations/008_assign_repair_id.sql

```diff
@@ -0,0 +1,14 @@
+--find max repair_id, then + 1
+CREATE OR REPLACE PROCEDURE assign_repair_id
```

Comment:
Should be a `FUNCTION` if it returns a value.
```suggestion
CREATE OR REPLACE FUNCTION assign_repair_id
```